schemas/service.yml

```diff
@@ -36,17 +36,32 @@
       description: >
         Name of the service data is collected from.
 
-        The name of the service is normally user given. This allows if two
-        instances of the same service are running on the same machine
-        they can be differentiated by the `service.name`.
+        The name of the service is normally user given. This allows for
+        distributed services that run on multiple hosts to correlate the
+        related instances based on the name.
 
-        Also it allows for distributed services that run on multiple hosts to
-        correlate the related instances based on the name.
-
-        In the case of Elasticsearch the service.name could contain the cluster
-        name. For Beats the service.name is by default a copy of the `service.type`
+        In the case of Elasticsearch the `service.name` could contain the cluster
+        name. For Beats the `service.name` is by default a copy of the `service.type`
         field if no name is specified.
 
+    - name: node.name
+      level: extended
+      type: keyword
+      example: "node1"
+      short: Name of the service node.
+      description: >
+        Name of a service node.
```

Comment:
I have a few things I'd like to be adjusted in this description, but I agree with the intent.

1. Please mention the use case for Elasticsearch nodes again. It's useful to have the same example reused across a field set. It will help cement the distinction between `service.name` and `service.node.name`. 
2. I like the thinking described in the last paragraph as well, as a general approach for multi-host applications that aren't specifically clustered (e.g. web apps in general). However right now it's phrased as APM documentation. Please phrase it more generally instead, here's an example:

> In cases where the service is not a clustered or doesn't have the concept of a node name, the host name or container name can be used to distinguish running instances that make up this service.

Reply:
I actually tried to follow the `service.name` description - tell the general use (first paragraph) and then provide examples (second paragraph). In the case of `service.name` the examples are Elasticsearch and Beats. 
I will add the Elasticsearch example and minimize the APM example description.